core/src/main/java/io/gnosis/kouban/core/ui/adapter/BaseAdapter.kt

```diff
@@ -33,6 +33,10 @@ class BaseAdapter<VH>(
     override fun onBindViewHolder(holder: VH, position: Int) {
         holder.bind(items[position])
     }
+
+    fun notifyAllChanged() {
+        notifyItemRangeChanged(0, items.size)
```

Comment:
why not simply notifyDataSetChanged

Reply:
They do different things if I remember correctly (should not matter much here). NotifyDataSetChanged will recreate all viewholders, while this one should just notify them to update themselves